src/stageCache.js

```diff
@@ -127,6 +130,7 @@ const updateStageCacheSettings = async (settings, serverless) => {
   }
 
   serverless.cli.log(`[serverless-api-gateway-caching] Updating API Gateway cache settings.`);
+  serverless.cli.log(`[serverless-api-gateway-caching] Updating the stage with: ${JSON.stringify(params)}`);
```

Comment:
I'd drop this log line because `params` could potentially be very large.

Reply:
Yes, I was using that for debugging reasons.